examples/meta/src/kernel/circular.sg

```diff
@@ -0,0 +1,19 @@
+File traindat = csv_file("../../data/fm_train_real.dat")
+File testdat = csv_file("../../data/fm_test_real.dat")
+
+#![create_features]
+Features feats_train = features(traindat)
+Features feats_test = features(testdat)
+#![create_features]
+
+#![create_kernel]
+Distance d = distance("EuclideanDistance", lhs=feats_train, rhs=feats_train)
```

Comment:
Can you mark left and right hand side parameters as read only? And then we force users to use the unit function, just like done in CKernel.
Sorry I didn’t say before we just added this option